.travis.yml

```diff
@@ -26,3 +36,7 @@ script:
     - OMP_NUM_THREADS=1 tests/src/eigenwave3d
     - python tests/eigenwave3d.py default --output
     - python tests/eigenwave3d.py read
+    - python tests/eigenwave3d.py pluto --execute --nthreads=4
+    - OMP_NUM_THREADS=4 tests/src/eigenwave3d_pluto
+    - python tests/eigenwave3d.py pluto --output
+    - python tests/eigenwave3d.py read
```

Comment:
Cool, please add the script to this branch if it is functional already. Does that mean you also have implemented the tile size switch for the example script?